mkosi.profiles/gnome-kiosk/mkosi.extra/usr/lib/sysusers.d/10-particleos-kiosk.conf

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+#Type Name       ID                  GECOS              Home directory Shell
+u     kiosk      -                   "Kiosk"            /home/kiosk    /sbin/nologin
```

Comment:
Is `/sbin/nologin` still the preferred way to encode this in Fedora? In Arch this works because `/sbin` is a symlink to `/usr/bin`. In Debian only this would work, though, because bin and sbin are not merged.

Reply:
Fedora doesn't merge `/bin` and `/sbin`, and puts `nologin` in `/sbin`. Looking at `/etc/passwd` on my system everything uses either `/sbin/nologin` or `/usr/sbin/nologin` (which are the same due to usrmerge).